libs/trades/src/lib/trades-container.tsx

```diff
@@ -6,26 +6,21 @@ import { AsyncRenderer } from '@vegaprotocol/ui-toolkit';
 import type { AgGridReact } from 'ag-grid-react';
 import { useCallback, useMemo, useRef } from 'react';
 import type { BodyScrollEvent, BodyScrollEndEvent } from 'ag-grid-community';
-import {
-  MAX_TRADES,
-  tradesDataProvider as dataProvider,
-} from './trades-data-provider';
+import { MAX_TRADES, tradesWithMarketProvider } from './trades-data-provider';
 import { TradesTable } from './trades-table';
-import type { TradeFields } from './__generated__/TradeFields';
 import type {
-  TradesVariables,
-  Trades_market_tradesConnection_edges,
-} from './__generated__/Trades';
+  TradeWithMarket,
+  TradeWithMarketEdge,
+} from './trades-data-provider';
+import type { TradesVariables } from './__generated__/Trades';
```

Comment:
chore (non blocking): This comment is applicable in quite a few places in the PR. We are often importing from the older generated types (`__generated__` rather than `__generated___`). This could certainly be done as a follow up task but it would be good to switch over if possible.

For example here we are using TradesVariables from the old generated types but later in trades-table.tsx we are using types from the new generated types.

Reply:
Definitely we want to switch to graphql-codegen style. But we did not wated to complicate this PR adding another change. This will be done postmerge